packages/ui/components/ui/box.tsx

```diff
@@ -34,19 +34,21 @@ export const Box = ({
   layout,
   layoutId,
   headerContent,
+  className,
 }: PropsWithChildren<
   VariantProps<typeof variants> & {
     label?: ReactNode;
     layout?: boolean;
     layoutId?: string;
     headerContent?: ReactNode;
+    className?: string;
```

Comment:
This has been one of the things that @jessepinho particularly has been hunting down to get rid of. By default, shadcn/ui takes a className-first kind of approach. This makes the consumer have more flexibility on passing any kind of style into the child. However, this also creates an API surface that is a bit more risky. Different parts our app can possibly customize too much. The solution we are converging toward is relying more upon variants and explicitly enumerated props passing.

In this case, we should find a way to accomplish the effect you are going for, but without requiring className as a prop.

Reply:
Right, got it. Changed to allow the `overflow` variant with type `xHidden` to the Box component, while removing className. Completely agree on the approach